docs/reference/samplers.rst

```diff
@@ -4,27 +4,21 @@
 Samplers
 ========
 
-.. currentmodule:: dwave.system.samplers
-
-.. contents::
-    :depth: 3
+A :term:`sampler` is the component used to find variable values that minimize the binary
```

Comment:
I think especially in the context of the `DWaveSampler` we might want to either reference the sampling aspect or just pass on the whole thing.

Something like "A :term:`sampler` accepts binary quadratic models and returns variable assignments. Samplers generally try to find minimizing values but can also sample from distributions defined by the BQM."

Reply:
Done